pom.xml

```diff
@@ -39,11 +39,11 @@ THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 
     <groupId>cloud.orbit</groupId>
     <artifactId>orbit-redis-cluster</artifactId>
-    <version>1.2.8-SNAPSHOT</version>
+    <version>1.2.9-SNAPSHOT</version>
```

Comment:
So this should likely be 1.2.8.4-SNAPSHOT since this was forked from 1.2.8 and we already have a 1.2.8.3

Reply:
Makes sense. I'm going to see what Joe says about the preferred version number in the PR I have open in the base project before making any changes.